.pinned

```diff
@@ -1,6 +1,6 @@
 bearssl;https://github.com/status-im/nim-bearssl@#acf9645e328bdcab481cfda1c158e07ecd46bd7b
 chronicles;https://github.com/status-im/nim-chronicles@#32ac8679680ea699f7dbc046e8e0131cac97d41a
-chronos;https://github.com/status-im/nim-chronos@#f7835a192b45c37e97614d865141f21eea8c156e
+chronos;https://github.com/status-im/nim-chronos@#d488db3324aed2f096a63a926e3f709d6975e4df
```

Comment:
We can't merge until the PR is merged, unfortunately: https://github.com/status-im/nim-chronos/pull/362